server/enums/strengthCategory.ts

```diff
@@ -0,0 +1,13 @@
+enum StrengthCategory {
```

Comment:
The enums are useful, not only for filters to map values to screen text (below), but also for validators (which we'll be adding shortly)